src/core/symbology-ng/qgsstylev2.cpp

```diff
@@ -135,6 +136,31 @@ bool QgsStyleV2::saveSymbol( QString name, QgsSymbolV2* symbol, int groupid, QSt
   return true;
 }
 
+void QgsStyleV2::exportSymbol( QString folder, QString name, QString format, QSize size )
```

Comment:
I thought about the same thing after I pushed it up. I will do that as it makes more sense.